src/lib/yang/snabb-softwire-v2.yang

```diff
@@ -392,374 +820,25 @@ module snabb-softwire-v2 {
           }
         }
       }
+
+      container binding-table-version {
```

Comment:
This will end up being a bit verbose: `/snabb-softwire/binding-table/binding-table-version/binding-table-version`.  What about removing `binding-table-`?  Also is the time zone defined for the date/time?

I worry a bit about feature creep here; I wonder if we can punt on some of these.  Dunno.